ObjectData[] data
string description
string[] fields
int64 timesteps